simple/sample/input/config.json

```diff
@@ -24,6 +24,14 @@
     "wikidataids.csv": {
       "entityType": "Country",
       "provenance": "Provenance1 Name"
+    },
+    "latlng_events.csv": {
+      "importType": "events",
```

Comment:
maybe skip importType as it can deduce from eventType?

Reply:
I think it's better to be explicit here to avoid confusion when we add more types of imports (like entities). So keeping this as-is.

Reply:
but the above ones should have `importType: "observations"` to match here?

Reply:
`observations` is default so didn't add it earlier. But you're right, should match it for consistency. Let me add it.

Reply:
Done.